docs/gatsby-config.js

```diff
@@ -623,7 +623,7 @@ const sections = [
         "node-operation/machine-existing-operator",
         "node-operation/db-encryption-existing-operator",
         "node-operation/monitoring-nodes",
-        "node-operation/full-observer-node",
```

Comment:
@vishalchangrani It's no longer necessary to edit this gatsby config, since we're not using the gatsby site anymore.